testsuite/classlibrary/TestCoreUGens.sc

```diff
@@ -341,7 +341,8 @@ test_bufugens{
 
 		// Copying data from b to c:
 		{
-			RecordBuf.ar(PlayBuf.ar(numchans, b, BufRateScale.ir(b), doneAction: 2), c, loop:0) * 0.1;
+			RecordBuf.ar(PlayBuf.ar(numchans, b, BufRateScale.ir(b), doneAction: 2), c, loop:0);
+            DC.ar(0);
```

Comment:
I's also OK to return `0` from a function that you play. You can also output a control rate signal. (The relevant source code is in `GraphBuilder`).